frontend/ui/widget/touchmenu.lua

```diff
@@ -219,6 +219,12 @@ function TouchMenuBar:init()
             for i, sep in ipairs(self.icon_seps) do
                 local current_icon = i == k - 1 or i == k
                 self.icon_seps[i].style = current_icon and "solid" or "none"
+                if i == #self.icon_seps - 1 then
```

Comment:
Lol, the screen of the Kobo Mini is so small that the home/exit icon is always on the right, so I won't notice any different.